easybuild/easyconfigs/o/OpenPGM/OpenPGM-5.2.122-GCCcore-12.2.0.eb

```diff
@@ -35,8 +35,14 @@ builddependencies = [
     ('Python', '3.10.8'),
 ]
 
+dependencies = [
+    ('Perl', '5.36.0'),
+]
+
 start_dir = 'pgm'
 
+preconfigopts = "find . -name \*.pl -type f | xargs -n 1 sed -i -e 's;#!/usr/bin/perl;#!/usr/bin/env perl;' && "
```

Comment:
```suggestion
preconfigopts = r"find . -name \*.pl -type f | xargs -n 1 sed -i -e 's;#!/usr/bin/perl;#!/usr/bin/env perl;' && "
```

To fix
```
./easybuild/easyconfigs/o/OpenPGM/OpenPGM-5.2.122-GCCcore-12.2.0.eb:44:31: W605 invalid escape sequence '\*'
```

Reply:
Ah didn't notice that one during testing